tackle-test-generator-unit/src/org/konveyor/tackle/testgen/core/extender/TestSequenceExtender.java

```diff
@@ -884,8 +995,35 @@ private boolean executeSequence(String sequenceID) {
             System.setErr(origSysErr);
         }
 	}
+	
+    private boolean isDeclaredException(String excp, Object mtdOrCnstr) {
+    	
+    	if (excp == null || mtdOrCnstr == null) {
+    		System.out.println(excp == null? "excp is null" : "method is null");
+    		return false;
+    	}
+    	
+    	System.out.println("Searching for exception "+excp);
```

Comment:
should this be a print statement (and appear in verbose mode output of CLI) or logging statment?